pkg/workflows/wasm/host/module.go

```diff
@@ -91,6 +66,14 @@ type DeterminismConfig struct {
 	Seed int64
 }
 
+type EmitLabeler interface {
+	// Emit sends a message to the labeler's destination.
+	Emit(string) error
+
+	// WithMapLabels sets the labels for the message to be emitted.  Labels are cumulative.
+	WithMapLabels(map[string]string) EmitLabeler
```

Comment:
the host works with maps from unmarshalled protos, so we have a different interface defined locally